app/app.py

```diff
@@ -4,144 +4,152 @@
 from flask_restful import Api
 from flask_restful import Resource
 
-app = Flask(__name__)
-api= Api(app)
+APP = Flask(__name__)
+API = Api(APP)
 
-users = {}
-query = []
-answer = []
-answers = {"answer" : answer}
-queries = {"query" : query}
+USERS = {}
+QUERY = []
+ANSWER = []
+ANSWERS = {"ANSWER" : ANSWER}
+QUERIES = {"QUERY" : QUERY}
 
 class Home(Resource):
+    """homepage"""
     def get(self):
+        """ get homepage"""
         return jsonify({"message":"welcome to stackoverflowlite, post or answer questions!"})
```

Comment:
You ought to have corrected this too to follow pep8 - space between `"message:":` and `"welcome ...`